assets/static/js/buzzer.js

```diff
@@ -264,6 +279,18 @@ function checkIfAddPartyFormComplete() {
   }
 }
 
+//function to handle when user updates party size in table.
+function updatePartySize(){
+    $(".dropdown li a").click(function(){
+    $(this).parents(".dropdown").find('.btn').html($(this).text() + ' <span class="caret"></span>');
```

Comment:
Can you fix the indentation here por favor. The body of code below the `$(".dropdown li a").click(function(){` should be indented in one